### DISCUSSION TITLE
Scprime storage

### DISCUSSION THREAD

**Discussion by 31trainman** [akash-network/community#519] [Integrations]
Iv been helping the scprime cloud storage team work to bring their cloud storage platform to akash network along with their mining software. We have also discussed running certain parts of their infrastructure on akash.

**Comment by JayRama** [comment]
**Proposal for Partnership Opportunity between Akash Network and Xa Net Services**

**Introduction:**
The demand for decentralized computing and storage solutions is on the rise. Akash Network, a decentralized compute network, and Xa Net Services (XNS), a leading provider of decentralized S3-compatible storage, both stand at the forefront of this revolution. We propose a strategic partnership to explore new opportunities and services to our respective user bases.

**Background:**
Akash Network boasts a decentralized compute network comprising approximately 75 providers, while XNS operates a decentralized S3-compatible network with over 50 petabytes of storage across more than 2000 nodes globally. These numbers are expected to grow rapidly over the coming months.


**Proposal:**
XNS proposes an invitation to all Akash Network providers to join our Partner program, for additional benefits and partner pricing. This integration will enable Akash providers to offer decentralized storage solutions to their customers for a fraction of the cost of AWS S3.

**Relayer:**
The XNS Relayer is the key to decentralized cloud storage, eliminating the need for caching servers or corporate owned satellites that lead to centralization in the wrong places. Based on Akash architecture and a global infrastructure of over 70 datacenter Partners, it makes sense for those who are interested to become XNS Partners and operate Relayers in their facilities. Every individual Relayer configures its own Virtual Private Datacenter of Storage Providers that fit the localized need. Scale up/Scale down is no longer an issue as the Relayer adds capacity with uploads automatically and is always goal seeking the highest performing and durable nodes. As XNS Partners, the Akash node operators receive special pricing, custom support, SLAs and ToS guidelines.

**Benefits:**
- Enhanced Value Proposition: Integration of XNS storage solutions enhances the offerings of Akash Network providers, providing added value to both providers and end-users.
- Cost Efficiency: XNS is 80% below AWS S3 charges for a service that is more performant and durable. XNS Partner pricing makes this even better.
- Scalability: With over 50 petabytes of storage available across the XNS network, providers can meet the growing and evolving storage needs of their users effectively.
- Global Reach: Leveraging the XNS extensive network of over 2000 nodes across various countries ensures global accessibility and reliability, catering to users worldwide with minimal latency.

**Next Steps:**
- Exploratory Discussions: Initiate discussions to delve deeper into partnership specifics and address any questions or concerns from both parties.
- Technical Integration: The XNS Relayer seamlessly integrates with Akash Network providers' infrastructure, requiring no additional setup or integration efforts. This allows providers to offer decentralized storage solutions to their customers without additional complexity.
- Pilot Implementation: Conduct a pilot phase to evaluate the performance and scalability of the integrated solution in real-world scenarios, gathering feedback for refinement.
- Full-scale Deployment: Upon successful completion of the pilot phase, proceed with full-scale deployment, enabling Akash Network providers to offer XNS storage solutions to their customers seamlessly.

**Conclusion:**
A potential partnership between Akash Network and Xa Net Services presents a promising opportunity to leverage the strengths of both platforms and deliver innovative solutions to the decentralized computing and storage market. We look forward to exploring these opportunities.

**Reply by brewsterdrinkwater** [reply]
hello. I am not sure if you were trying to make this it's own discussion, or if you wanted it inside of this discussion. 

cc: @JayRama

**Reply by 31trainman** [reply]
I told him to post it on my thread as it was already existing on here.
